internal/pkg/image/driver/imagedriver.go

```diff
@@ -232,7 +232,7 @@ func (d *fuseappsDriver) Mount(params *image.MountParams, mfunc image.MountFunc)
 		}
 
 	case "encryptfs":
-		return fmt.Errorf("mounting an encrypted filesystem requires root or a suid installation")
+		return fmt.Errorf("reading a filesystem encrypted by root requires root or a suid installation")
```

Comment:
In re-reading that message I suggested it occurs to me that it is still a bit ambiguous.  I now suggest
```
reading a root-encrypted SIF requires root or a suid installation
```

Reply:
resolved bb0a3b5c547a0bf5c062437bf2c4e23202f45fbf